src/subscription_manager/intentstore_interface.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function, division, absolute_import
+
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# Red Hat trademarks are not licensed under GPLv2. No permission is
+# granted to use or replicate Red Hat trademarks that are incorporated
+# in this software or its documentation.
+#
+
```

Comment:
I prefer to have try-except statement with importing intentctl module here. Something like this:

```python
try:
    from intentctl.intentfiles import IntentStore, USER_INTENT
except ImportError:
    print("WARNING: Could not import from module intentctl. Intent SLA value not saved locally.")
```

Try to avoid importing of mudules/packages inside functions/clases.

Please do not print this information to console, but write it to log file using `log.erro()` (see my later comment about logging). it is not so critical information for user and it is IMHO not necessary to print it to console.

Reply:
I did initially use the form `from intentctl.intentfiles import IntentStore, USER_INTENT`, but I changed it because the unit test where '..module_does_not_exist' was [failing in python3.6](https://bugs.python.org/issue31642) when using `sys.modules['intent'] = None` to temporarily un-import the module.

Reply:
Oh. I didn't know about this. Good point. Anyway, I would not change source code to make implementation of unit tests easier. If this unit test does not work for Python 3 due to bug in Python 3.6/3.7, then you can skip this test for Python 3 using something like this:

```python
if not six.PY2:
    raise SkipTest("Skipping this unit test due to bug in Python 3.")
```

Reply:
That's helpful, I'll keep it in mind. I've found another way to test this though.